rasa_sdk/cli/arguments.py

```diff
@@ -54,3 +54,8 @@ def add_endpoint_arguments(parser):
         help="Enable auto-reloading of modules containing Action subclasses.",
         action="store_true",
     )
+    parser.add_argument(
+        "--endpoints",
+        default=DEFAULT_ENDPOINTS_PATH,
+        help="Configuration file for tracing as a yml file.",
```

Comment:
Minor comment: I expect the assistant endpoints file would be used which contains more details than just tracing config. I'd rephrase this to reflect that, unless you recommend separating the tracing config in a diff config file, in which case we should rename the arg?